pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -422,17 +422,20 @@ def make_package(args):
         )
 
     # Find the SDK directory and target API
-    with open('project.properties', 'r') as fileh:
-        target = fileh.read().strip()
-    android_api = target.split('-')[1]
-    try:
-        int(android_api)
-    except (ValueError, TypeError):
-        raise ValueError(
-            "failed to extract the Android API level from " +
-            "build.properties. expected int, got: '" +
-            str(android_api) + "'"
-        )
+    if 'ANDROIDAPI' in os.environ:
+        android_api = os.environ['ANDROID_API']
+    else:
+        with open('project.properties', 'r') as fileh:
```

Comment:
I kept this code in place but I'm not sure if there will ever be a case where `ANDROIDAPI` is not set during a build. Also, are `project.properties` included or used anywhere in gradle builds?